warehouse/integrations/snowflake/snowflake.go

```diff
@@ -450,6 +450,21 @@ func (sf *Snowflake) mergeIntoLoadTable(
 		updateSet = fmt.Sprintf(`original.%[1]q = original.%[1]q`, strKeys[0])
 	}
 
+	var mergeWindowClause string
+	configKeyPrefix := "Warehouse.snowflake.mergeWindow." + sf.Warehouse.Destination.ID
+	mergeWindowTables := sf.conf.GetStringSlice(configKeyPrefix+".tables", nil)
+
+	if slices.Contains(mergeWindowTables, tableName) {
+		mergeWindowDuration := sf.conf.GetDuration(configKeyPrefix+".duration", 24, time.Hour)
```

Comment:
1 day seems too low, Since by default we do dedup for 14 days in the processor. Shall we make it to 30 days by default?